### DISCUSSION TITLE
Can I run a swagger UI for my firefly-iii instance?

### DISCUSSION THREAD

**Discussion by Sleepful** [firefly-iii/firefly-iii#9116] [Firefly III] [answered]
Searched through docs and issues but didn't find mention of this.

**Comment by JC5** [comment, marked as answer]
Clone this repository and run it locally: https://github.com/firefly-iii/api-docs

**Comment by vstrimaitis** [comment]
Is there any chance to ship it together with the app's Docker image? I.e. the same version of the docs as that of the image, and it seems like it would be quite convenient to just go to `my-firefly-instance.domain.whatever/docs` and inspect the API 😁
